reference/3.0/Microsoft.PowerShell.Core/Update-Help.md

```diff
@@ -1,54 +1,62 @@
 ---
-ms.date:  06/09/2017
+ms.date:  5/16/2019
 schema:  2.0.0
 locale:  en-us
 keywords:  powershell,cmdlet
 online version:  http://go.microsoft.com/fwlink/?LinkID=210614
 external help file:  System.Management.Automation.dll-Help.xml
 title:  Update-Help
 ---
+
 # Update-Help
 
 ## SYNOPSIS
-
 Downloads and installs the newest help files on your computer.
 
 ## SYNTAX
 
 ### Path (Default)
 
 ```
-Update-Help [[-Module] <String[]>] [[-SourcePath] <String[]>] [-Recurse] [[-UICulture] <CultureInfo[]>]
- [-Credential <PSCredential>] [-UseDefaultCredentials] [-Force] [<CommonParameters>]
+Update-Help [[-Module] <string[]>] [[-SourcePath] <string[]>] [[-UICulture] <cultureinfo[]>]
+[-Recurse] [-Credential <pscredential>] [-UseDefaultCredentials] [-Force] [<CommonParameters>]
 ```
 
 ### LiteralPath
 
 ```
-Update-Help [[-Module] <String[]>] [-LiteralPath <String[]>] [-Recurse] [[-UICulture] <CultureInfo[]>]
- [-Credential <PSCredential>] [-UseDefaultCredentials] [-Force] [<CommonParameters>]
+Update-Help [[-Module] <string[]>] [[-UICulture] <cultureinfo[]>] [-LiteralPath <string[]>]
+[-Recurse] [-Credential <pscredential>] [-UseDefaultCredentials] [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-The `Update-Help` cmdlet downloads the newest help files for PowerShell modules and installs them on your computer.
-You can use the `Get-Help` cmdlet to view the new help files immediately.
-You do not have to restart PowerShell to make the change effective.
-This feature enables you to install help files for modules that do not include them and to update help files on your computer so that they never become obsolete.
+The `Update-Help` cmdlet downloads the newest help files for PowerShell modules and installs them on
+your computer. You can use the `Get-Help` cmdlet to view the new help files immediately. You don't
+need to restart PowerShell to make the change effective.
+
+`Update-Help` checks the version of the help files on your computer. If you don't have help files
+for a module or don't have the newest help files for a module, `Update-Help` downloads the newest
+help files from the internet or a file share and installs them on your computer in the correct
+module folder.
 
-`Update-Help` checks the version of the help files on your computer.
-If you do not have help files for a module or do not have the newest help files for a module, `Update-Help` downloads the newest help files from the Internet or a file share and installs them on your computer in the correct module folder.
+Without parameters, `Update-Help` updates the help files for modules in the session and for all
+installed modules that support Updatable Help. Modules that aren't in the current session are
```

Comment:
I think we need to be clearer here:

Modules that are installed but not loaded in the current session are included.

Reply:
Resolved in updated doc.